### PR TITLE
Use different application id, app name and icon for debug builds

### PR DESCRIPTION
**Pull request by Maxr1998**
I tested this locally and it seems to be working fine. I think we don't use the package name hardcoded anywhere anyway.

### CODE REVIEW

**Comment by nielsvanvelzen** [comment]
Can we use the debug icon for debug builds? (it's the orange one)

**Comment by Maxr1998** [comment]
> Can we use the debug icon for debug builds? (it's the orange one)

There is a debug icon? But yeah, sure. Just need to find the resources.

File: app/build.gradle.kts [hunk 1]
```diff
@@ -12,7 +12,7 @@ plugins {
 android {
     compileSdkVersion(30)
     defaultConfig {
-        applicationId = "org.jellyfin.mobile"
```
Comment (nielsvanvelzen) [inline app/build.gradle.kts:15]: For ATV we only append .debug on debug, I think we should mirror that behavior

Comment (nielsvanvelzen) [reply app/build.gradle.kts:15]: (so always include the .mobile part)

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (Maxr1998) [reply app/build.gradle.kts:15]: I decided against that because the package name then gets quite long and gets a bit weird..

Comment (nielsvanvelzen) [reply app/build.gradle.kts:15]: While I get your point I still think it's a bad decision. "org.jellyfin" is basically our prefix while "mobile" is the name for this specific app and "androidtv" is the tv app.

The ".debug" part could be interpreted as a flavor (although not as "android flavor") which is added only for non-default builds.

Comment (Maxr1998) [reply app/build.gradle.kts:15]: Hmm, that argument makes sense. I'm still not 100% happy with the alternative. Maybe `.dbg` or `.dev` as suffix to keep it shorter?

Comment (nielsvanvelzen) [reply app/build.gradle.kts:15]: We could do that but I'd like to keep this consistent with the ATV app so we'd have to change it there too.

Comment (dkanada) [reply app/build.gradle.kts:15]: Appending `.debug` is pretty common from what I've seen, I think that would be fine.

Comment (Maxr1998) [reply app/build.gradle.kts]: Fine, changed it.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**